stack-graphs/src/stitching.rs

```diff
@@ -757,6 +764,7 @@ impl<H> ForwardPartialPathStitcher<H> {
         _graph: &StackGraph,
         _partials: &mut PartialPaths,
         initial_partial_paths: I,
+        config: &StitcherConfig,
```

Comment:
I think this will make more sense to pass by value instead of by reference, since it's a plain ol' data type (and currently smaller in size than a pointer anyway!)